content/release/ReleaseNotes373.md

```diff
@@ -0,0 +1,156 @@
+<!--
+title: "Contrast 3.7.3 - April 2020"
+description: "Contrast 3.7.3 April 2020"
+tags: "3.7.3 April Monthly Release News"
+-->
+
+# Monthly Release News - April 2020
+
+This page shows highlights from our all Contrast hosted, on-premises and agent releases over the past month.
+
+**3.7.3 on-premises release date: ** May 5, 2020
+
+**New and improved:**
+
+- In addition associating vulnerabilities, you can now also **associate both discovered and exercised routes** to build numbers, application versions, branches or repositories **using session metadata**. This means you can also query route information with a public API endpoint. With a single call to a public endpoint you can get detailed information on how much of an application has been exercised and where the critical vulnerabilities are.
+- You now have a choice to receive individual policy violation emails or to consolidate them into a single email. Find this option under **Organization Settings \&gt; Notifications**.
```

Comment:
Not sure if `>` was supposed to be escaped twice here.

Reply:
I think you are talking about the clickpath, which should just have one >